source/components/conversation/RuleInput.tsx

```diff
@@ -52,8 +53,21 @@ export const binaryQuestion = [
 	{ value: 'non', label: 'Non' },
 ] as const
 
-export const isMosaic = (dottedName) =>
-	mosaicQuestions.find(({ isApplicable }) => isApplicable(dottedName))
+export const isMosaic = (engine, rules, dottedName) => {
```

Comment:
I am not sure ton understand your last suggestion ?

I tried to return `undefined` but I had an error when calling the function with an array because of the undefined. it was expected an empty array.. Maybe I didn't manage to do it properly when returning `undefined`..

Reply:
I suggested to modify the return type of the function to explicitly express that there are two possible cases: the question is a mosaic or not.
It can be done in different ways, but the simpliest is to return undefined if no information about the mosaic was found.